lib/gnat/aunit_shared.gpr

```diff
@@ -34,6 +35,11 @@ project AUnit_Shared is
          Calend := "nocalendar";
          Memory := "nodealloc";
          FileIO := "nofileio";
+      when "ravenscar-sfp" =>
```

Comment:
How is this case different from the Aunit zfp profile? AUnit doesn't use tasking so from its point of view ravenscar-sfp is no different from zfp.

Reply:
You're right, I fixed that in https://github.com/AdaCore/aunit/pull/33/commits/bcf106bf9f4b6ade59e96074a9dad7555b7806f4.